src/bzfs/Makefile.am

```diff
@@ -17,7 +17,12 @@ else
 plugin_files =
 endif
 
+if APPLE
+bzfs_LDFLAGS = -export-dynamic -static -framework CoreFoundation -framework Carbon
```

Comment:
I actually really don't like this; let me replace this with entries in `configure.ac` (ditto for the other two Makefiles)